doc/3/controllers/document/index.md

```diff
@@ -0,0 +1,55 @@
+---
+code: true
+type: page
+title: create
+description: document:create
+---
+
+# create
+
+Creates a new document in the provided index and collection.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> create(
```

Comment:
The create method should have at least 2 signature, one with the `id` parameter and the other one without so Kuzzle can generate an ID